mmv1/products/iam3/PrincipalAccessBoundaryPolicy.yaml

```diff
@@ -41,12 +46,20 @@ async:
   include_project: true
 examples:
   - name: 'iam_principal_access_boundary_policy'
-    primary_resource_id: 'my-pab-policy'
+    primary_resource_id: 'pab-policy-for-org'
     test_env_vars:
       org_id: 'ORG_ID'
     vars:
-      display_name: 'test pab policy'
-      pab_id: 'test-pab-policy'
+      display_name: 'PAB policy for Organization'
+      pab_id: 'pab-policy-for-org'
+  - name: 'iam_organizations_policy_binding'
+    primary_resource_id: 'binding-for-all-org-principals'
```

Comment:
This is the test that's failing. In this case the test is looking for a PAB policy with the id `binding-for-all-org-principals` but in the [iam_organizations_policy_binding template](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/iam_organizations_policy_binding.tf.tmpl) the value is hard coded as `pab_policy`. That said, even if the example were updated it wouldn't be testing anything that isn't already tested in `iam_principal_access_boundary_policy` and the example is still available under OrganizationsPolicyBinding so it's not really adding anything. If you remove this new example block the tests should pass.